suitcase/mongo_normalized/__init__.py

```diff
@@ -90,6 +92,24 @@ def __call__(self, name, doc):
         sanitized_doc = event_model.sanitize_doc(doc)
         return super().__call__(name, sanitized_doc)
 
+    def update(self, name, doc):
+        if name in ['start', 'stop']:
```

Comment:
Our conversation in group meeting reminded me of two things:

1. The request from CHX was that "all information directly typed by the user should be editable." We don't need to support editing `'stop'` documents in order to satisfy that.
2. Allowing users to edit `'stop'` document is could enable them to (accidentally or on purpose) change which `'start'` document they refer to and thus put things out of sync. This is the same problem we would have with all the other document types. For now, let's limit it to `'start'` only in order to keep things simple.